Templates/Swift/Sources/APIClient.swift

```diff
@@ -151,7 +155,10 @@ public class APIClient {
         switch dataResponse.result {
         case .success(let value):
             do {
-                let statusCode = dataResponse.response!.statusCode
+                guard let statusCode = dataResponse.response?.statusCode else {
```

Comment:
While it is uncommon that there has been no error but also no response, it is definitely possible this can happen. Let's be sensible and avoid crashing.